mod/admin.php

```diff
@@ -908,6 +908,7 @@ function admin_page_summary(App $a)
 
 	$queues = ['label' => L10n::t('Message queues'), 'queue' => $queue, 'workerq' => $workerqueue];
 
+	$php_settings = [L10n::t('PHP Values'), ['upload_max_filesize' => ini_get('upload_max_filesize'), 'post_max_size' => ini_get('post_max_size'), 'memory_limit' => ini_get('memory_limit')]];
```

Comment:
Please use an associative array withe the `title` and `values` keys. Make sure to update the references to `$php` in the templates.